stdlib/CompilerSupportLibraries_jll/src/CompilerSupportLibraries_jll.jl

```diff
@@ -76,7 +76,12 @@ const libgfortran = LazyLibrary(_libgfortran_path, dependencies=_libgfortran_dep
 _libstdcxx_dependencies = LazyLibrary[libgcc_s]
 const libstdcxx = LazyLibrary(_libstdcxx_path, dependencies=_libstdcxx_dependencies)
 
-const libgomp = LazyLibrary(_libgomp_path)
+if Sys.iswindows()
+    _libgomp_dependencies = LazyLibrary[libgcc_s]
+else
+    _libgomp_dependencies = LazyLibrary[]
+end
+const libgomp = LazyLibrary(_libgomp_path, dependencies=_libgomp_dependencies)
```

Comment:
Do we need to store in the pkgimage useless non-const global variables?  Looks like this is done everywhere in the JLLs now, but seems a waste.

Reply:
Fair point. I’ll fix that